app/assets/javascripts/models/ui/state_manager.coffee

```diff
@@ -247,10 +248,12 @@
         @_sendToCollectionList()
         path = @path()
         serialized = @serialize()
+        refreshColList = true
 
       if @_isValid(path, serialized)
         changed = urlUtil.saveState(path, serialized, !@historyChanged, @page.workspaceNameField.peek())
         @historyChanged = true if changed
+        @page.collections.isRelevant(refreshColList)
```

Comment:
This should be:

    @page.collections.isRelevant(true) if refreshColList

It's possible that collections are relevant when you get to this point, and you don't want them to accidentally be set to false. Given that, I'm not actually sure how the specs all seem to be passing.